web_responsive/readme/DESCRIPTION.rst

```diff
@@ -103,16 +60,6 @@ This module adds responsiveness to web backend.
 
   .. image:: ../static/img/formview.gif
 
-* Set chatter on the side of the screen, optional per user
-
-  .. image:: ../static/img/chatter_sided.gif
-
```

Comment:
Why has this been deleted?

Reply:
As this feature(setting chatter position) would be moved from this module to another module (web_chatter_position) as suggested by @pedrobaeza in https://github.com/OCA/web/issues/2336